subprojects/detekt.yml

```diff
@@ -0,0 +1,689 @@
+build:
+    maxIssues: 0
+    excludeCorrectable: false
+    weights:
+    # complexity: 2
+    # LongParameterList: 1
+    # style: 1
+    # comments: 1
+
+config:
+    validation: true
+    warningsAsErrors: false
+    # when writing own rules with new properties, exclude the property path e.g.: 'my_rule_set,.*>.*>[my_property]'
+    excludes: ''
+
+processors:
+    active: true
+    exclude:
+        - 'DetektProgressListener'
+    # - 'FunctionCountProcessor'
+    # - 'PropertyCountProcessor'
+    # - 'ClassCountProcessor'
+    # - 'PackageCountProcessor'
+    # - 'KtFileCountProcessor'
+
+console-reports:
+    active: true
+    exclude:
+        - 'ProjectStatisticsReport'
+        - 'ComplexityReport'
+        - 'NotificationReport'
+        #  - 'FindingsReport'
+        - 'FileBasedFindingsReport'
+
+output-reports:
+    active: true
+    exclude:
+    # - 'TxtOutputReport'
+    # - 'XmlOutputReport'
+    # - 'HtmlOutputReport'
+
+comments:
+    active: false
```

Comment:
All is copied from detekt default config and disabled;

Proposed pattern is to enable and fix valuable(for us) rules one by one and not use baseline mechanism

Reply:
How about adding this knowledge to this file?

Also, it'd be helpful to add a link to the reference: 
https://github.com/detekt/detekt/blob/master/detekt-core/src/main/resources/default-detekt-config.yml